developer/components/icon.md

```diff
@@ -12,20 +12,20 @@ The icon components give you a simple way to use the available icon sets in Reac
 
 ###### Basic Usage
 
-```js
-import { icon } from "meteor/reactioncommerce:reaction-ui";
+```
```

Comment:
Code block language is missing here (linter should've caught that - let me know if it didn't for you)